packages/sdk-demo/src/context/configContext/configContext.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { createContext, useContext, memo } from 'react';
+
+import { getConfig, ConfigSchema } from '@/core/getConfig';
+
+type ConfigContextType = ConfigSchema | null;
+
+const ConfigContext = createContext<ConfigContextType>(null);
+
+interface ConfigProviderProps {
+  config: ConfigSchema;
+  children: React.ReactNode;
+}
+
+const ConfigProviderComponent = memo(
```

Comment:
`memo(...)` is not needed here, as `config` and `children` are passed directly to the context. `memo` is needed to optimize the component's logic computing.

Reply:
fixed